src/main/java/io/strimzi/kafka/bridge/mqtt/kafka/BridgeKafkaProducerService.java

```diff
@@ -4,54 +4,88 @@
  */
 package io.strimzi.kafka.bridge.mqtt.kafka;
 
-import io.netty.handler.codec.mqtt.MqttQoS;
 import io.strimzi.kafka.bridge.mqtt.config.KafkaConfig;
 import io.strimzi.kafka.bridge.mqtt.utils.KafkaProducerAckLevel;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
 import org.apache.kafka.clients.producer.ProducerRecord;
 import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.common.serialization.ByteArraySerializer;
+import org.apache.kafka.common.serialization.StringSerializer;
 
+import java.util.Properties;
+import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
 
 /**
- * A service to handle the Kafka Producers
+ * Represents a Kafka producer for the Bridge.
  */
 public class BridgeKafkaProducerService {
```

Comment:
Can we stick with what we have in the HTTP bridge for the name, it will be simpler to deal with comparisons. I would say it could be just `KafkaBridgeProducer`.